### ISSUE TITLE
Resolve Fractal Manager from the IoC

### ISSUE THREAD

**Issue by prYmea** [yajra/laravel-datatables#240]
Hi,

Is it possible to get an instance of the Fractal manager from the IoC in the `render` method (`BaseEngine.php` line 663) like below:

```
   $fractal = app(\League\Fractal\Manager::class);
```

Reason:
That way I can bind the fractal manager in a custom service provider to set a serializer. At the moment there is no way (correct me if I am wrong) to change the transformer serializer in your package.

Thanks!

**Comment by yajra** [comment]
Thanks for bringing this up. I haven't used much of fractal's features and not yet sure on serializer. Will try to implement this on my free time. Or you can also send a PR if you can, since I think you know Fractal more than I do. ^_^ Thanks!

**Comment by prYmea** [comment]
I have submitted a pull request with the simple changes that I mentioned in my original post.

Using fractal [serializer](http://fractal.thephpleague.com/serializers/) one can change the structure of the transformer's output. 

I created the below custom service provider where I can swap out the serializer to a different one or even a custom one if I wanted.

```
use Illuminate\Support\ServiceProvider;

class FractalServiceProvider extends ServiceProvider
{
    /**
     * Register the application services.
     *
     * @return void
     */
    public function register()
    {
        $this->app->bind(\League\Fractal\Manager::class, function($app)
        {
            $fractal = new \League\Fractal\Manager;

            $serializer = new \League\Fractal\Serializer\ArraySerializer();

            $fractal->setSerializer($serializer);

            return $fractal;
        });

        $this->app->bind(\Dingo\Api\Transformer\Adapter\Fractal::class, function($app)
        {
            $fractal = $app->make(\League\Fractal\Manager::class);

            return new \Dingo\Api\Transformer\Adapter\Fractal($fractal);
        });
    }
}
```

**Comment by yajra** [comment]
Should be fixed on v5.12.0. Thanks!
